src/tests/base-tests-utils.ts

```diff
@@ -0,0 +1,5 @@
+import { ComponentDefaultTestId as TestIds } from "./constants";
+export const ComponentDefaultTestId = TestIds;
+export const getTestId = (elementType: TestIds, id?: string | number) => {
```

Comment:
Why we decided to move to that instead of `ELEMENT_TYPES`? 
I'm afraid that'll make Vibe 2.0 branch support even harder :(